frontend/src/admin/createUser/CreateUserPageContents.jsx

```diff
@@ -1,20 +1,12 @@
 import React from "react";
-
-import FormFailedDialog from "../../general/userForm/FormFailedDialog";
+import FormConsole from "../../general/userForm/FormConsole";
 import { createUser } from "../../api/UserManagementApi";
-import CreateUserFormConsole from "./CreateUserFormConsole";
 import { snackbar } from "../../general/SnackbarMessage";
 
 export default class CreateUserPageContents extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {
-      dialogMessage: ""
-    };
-    this.dialogElement = React.createRef();
     this.handleSubmit = this.handleSubmit.bind(this);
-    this.openDialog = this.openDialog.bind(this);
-    this.setDialogMessage = this.setDialogMessage.bind(this);
   }
 
   handleSubmit(username, password) {
```

Comment:
Can you get rid of the code smell? Perhaps by moving the function out of the class.

Reply:
8659c3c